classes/generic_configline.php

```diff
@@ -67,15 +67,15 @@ public function get_rubrics() {
                     current($categories)->coursecount > $this->config->maxallcourse)) {
                 $label = $this->line['catlistheading'];
                 $list = $categories;
-                $this->rubrics[] = new block_filtered_course_list_rubric($label, $list, $this->config, $this->line['expanded']);
+                $this->rubrics[] = new \block_filtered_course_list_rubric($label, $list, $this->config, $this->line['expanded']);
             } else {
                 // Just print course names of single category.
                 $category = array_shift($categories);
                 $courses = get_courses($category->id);
                 if ($courses) {
                     $label = $this->line['courselistheading'];
                     $list = $courses;
-                    $this->rubrics[] = new block_filtered_course_list_rubric($label, $list, $this->config, $this->line['expanded']);
+                    $this->rubrics[] = new \block_filtered_course_list_rubric($label, $list, $this->config, $this->line['expanded']);
```

Comment:
Line 78 ends up being one character too long for the code checker. I know this is not the only warning that needs to be cleaned up, but this one is directly related to the current change, so please shorten or split the line.